shared-module/displayio/ColorConverter.c

```diff
@@ -128,9 +129,29 @@ bool common_hal_displayio_colorconverter_get_dither(displayio_colorconverter_t*
     return self->dither;
 }
 
+void common_hal_displayio_colorconverter_make_transparent(displayio_colorconverter_t* self, uint32_t transparent_color) {
+    if (self->transparent_color) {
```

Comment:
As is, I think this will fail if the transparent color is black.

```suggestion
    if (self->transparent_color >= 0x1000000) {
```